src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Excavator is working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator has stopped");

```

Comment:
The message in the `stopWork` method should be more informative by indicating the type of `Machine` that has stopped, such as 'Excavator has stopped working'.